### ISSUE TITLE
Keyboard Navigation Accessibility Issue: Unable to Open Item in new brukerflate GUI via Keyboard Interaction

### ISSUE THREAD

**Issue by sneha-sirure** [Altinn/altinn-access-management-frontend#1219]
Det som jeg prøver å bruke bare keyboard for navigering i det nye brukerflate GUI, men etter jeg trykket på tilgangspakke, kunne ikke åpne det via "Enter" key uten å bruke mouse. Det skjer samme etter man går inne i "Gi fullmakter til ny tilgangspakke" da skjer ingenting når prøver å åpne tilgangspakker via keyboard (Enter key)

![Image](https://github.com/user-attachments/assets/e8c8c7ad-8a2e-4cbf-8e9c-405421c00be8)

**Comment by allinox** [comment]
@sneha-sirure

**Comment by allinox** [comment]
@sneha-sirure 
You should now be able to open expandable items using your keyboard in AT22.
There is a quirk with the items where the tab order is lost when opening the item (so you will have to tab one extra time to get back to the closing button). This is something we are aware of and we are planning on fixing in the [altinn-components](https://github.com/Altinn/altinn-components) library.

**Comment by sneha-sirure** [comment]
@allinox I have verified this and looks good so closing taks. good job :)
